### DISCUSSION TITLE
DAI 2024-2025 - Practical work 1 - Yoann Changanaqui and Camille Theubet

### DISCUSSION THREAD

**Discussion by Yoy017** [heig-vd-dai-course/heig-vd-dai-course#327] [Show and tell]
A deux, nous allons créer un projet à l'aide de la librairie picocli. Cette librairie nous permet de créer nos propres ligne de commandes personnaliser. On imaginera un projet qui traitera différents fichiers par là suite !

**Comment by GeraudSilvestri** [comment]
Il nous faut plus de détails sur les fonctionnalités supplémentaires de votre projet. Il est bien de les poser avant de commencer à coder.

**Comment by Yoy017** [comment]
Our project is a task list (todo list) manager. It would allow the user to create list, add tasks to lists, categories of the tasks, sort them depending on the categories or the deadlines, add a description and much more.

**Reply by ludelafo** [reply]
It's an excellent idea! Please be wary about the available time, you might want to reduce the scope of your idea by only allowing to add/remove items to/from a list and to sort them. It would already be enough for me.

Feel free to start small and add more features as you go if you have the time.

Do you agree @GeraudSilvestri?

**Reply by GeraudSilvestri** [reply]
It seems good to me with @ludelafo's comment

**Comment by CamilleTCH** [comment]
Hello,

Link to our repository : [todo-manager](https://github.com/Yoy017/todo-manager)
Here is the commit hash for the evaluation : 0057e4f2307a23c3c396426803fdff3c2cf9e613 [Link to hash](https://github.com/Yoy017/todo-manager/commit/0057e4f2307a23c3c396426803fdff3c2cf9e613)
As a bonus : tag to the final commit : https://github.com/Yoy017/todo-manager/releases/tag/Final_commit

Looking forward for you to discover todo-manager :o. See you tomorow !
